docs/Config_Reference.md

```diff
@@ -4652,6 +4652,15 @@ host_mcu:
 #   (True sets CFG5 high, False sets it low). The default is True.
 ```
 
+### [ina381]
+The INA381 is an current monitor chip with a programmable over current
+alarm.
+
+This chip needs a reset pulse in order to operate in latching mode,
+so that is all this plugin does. The rest of the functionality can be
+implemented using already existing functionality in Klipper.
```

Comment:
The config reference would need to describe all the available config options (eg, `reset_pin`) and their defaults.

Reply:
It would be great to be able to fix this without requiring a custom module. I've done some tests with this suggestion and I was not able to get delayed_gcode to work, but after thinking about this a bit I think its just as good to let this be a macro button that the user can press if the over current alarm is tripped. I think that is safer. I will try to implement this in the config files and resubmit.